app/controllers/users/service_provider_inactive_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Users
+  class ServiceProviderInactiveController < ApplicationController
+    include FullyAuthenticatable
```

Comment:
the fully authenticatable bit confuses me... do we need it? my understanding is that this is essentially a logged-out message

Reply:
I pulled in the FullyAuthenticatable for the delete_branded_experience, since the show action is for redirecting from the SP branded error page to the non-SP branded login page. However, after playing around with it, I'm not 100% certain if that is needed in this case. I'm not clear on what is deleted from the Redis pool and what effect that has.

Reply:
Ok, after chatting with Steve a little I think it makes sense to be using the `delete_branded_experience` method but I realized I could simplify that whole redirect thing, now that I understand better what it's doing.

Reply:
Though, we may want to rename `FullyAuthenticatable` since that doesn't really seem to be convey the purpose of the module/mixin very well.